<launch>
  <arg name="slam"/>
  <include if="$(arg slam)" file="$(find gsm_node)/launch/rovioli.launch"/>

  <include file="$(find gsm_node)/launch/gsm_with_depth_segmentation_monstah.launch">
    <arg name="sliding_window" default="true"/>
    <arg if="$(arg slam)" name="camera_frame" value="camera0"/>
    <arg if="$(arg slam)" name="imu_frame" value="imu"/>
  </include>

</launch>
